packages/video_player/video_player/lib/video_player.dart

```diff
@@ -479,6 +483,25 @@ class VideoPlayerController extends ValueNotifier<VideoPlayerValue> {
     await _videoPlayerPlatform.setLooping(_textureId, value.isLooping);
   }
 
+  void _applyUpdateDurationPeriodic() {
```

Comment:
他の同様のapplyメソッドでは、先頭で```_isDisposedOrNotInitialized```のチェックしていますが、同じくここでも必要な雰囲気ありませんか

```suggestion
  void _applyUpdateDurationPeriodic() {
    if (_isDisposedOrNotInitialized) {
      return;
    }
```

Reply:
ありがとうございます。
修正しました！